scriptmodules/supplementary/scraper.sh

```diff
@@ -47,7 +47,7 @@ function scrape_scraper() {
     params+=(-output_file "$gamelist")
     params+=(-rom_dir "$romdir/$system")
     params+=(-workers "4")
-    [[ "$system" =~ ^mame- ]] && params+=(-mame -mame_img t,m,s)
+    [[ "$system" =~ ^mame- ]] || [[ "$system" =~ ^fba ]] || [[ "$system" =~ ^neogeo ]] || "$system" =~ ^arcade ]] && params+=(-mame -mame_img t,m,s)
```

Comment:
We only use ^mame- to match all the rom folders prefixed with mame - if you want to use others just either match individual strings or use a regular expression with |

`[[ "$system" =~ ^mame- || "$system" =~ fba|neogeo|arcade ]]`